common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -669,6 +696,11 @@ public String getID() {
       return streamId;
     }
 
+    @Override
+    public ByteBuffer getCompletionResponse() {
+      return new BlockPushReturnCode(ReturnCode.SUCCESS.id()).toByteBuffer();
+    }
+
```

Comment:
nit:
Given how frequently this will be called - make this into a static final read only bytebuffer we duplicate each time it is required ?

Something like:

```
    private static final ByteBuffer SUCCESS_BUFFER =
            new BlockPushReturnCode(ReturnCode.SUCCESS.id()).toByteBuffer().asReadOnlyBuffer();
```

and getCompletionResponse -> `SUCCESS_BUFFER.duplicate()`